eng/Versions.props

```diff
@@ -135,6 +135,7 @@
     <XunitPerformanceApiPackageVersion>1.0.0-beta-build0015</XunitPerformanceApiPackageVersion>
     <MicrosoftDiagnosticsToolsRuntimeClientVersion>1.0.4-preview6.19326.1</MicrosoftDiagnosticsToolsRuntimeClientVersion>
     <MicrosoftDiagnosticsNETCoreClientVersion>0.2.61701</MicrosoftDiagnosticsNETCoreClientVersion>
+    <NuGetBuildTasksPackageVersion>5.10.0-preview.2.7203</NuGetBuildTasksPackageVersion>
```

Comment:
Can we update line 147 to this version instead of a new entry? `RefOnlyNugetProjectModelVersion` is used here: https://github.com/dotnet/runtime/blob/28b9c72d25c3a58e85d08bc1ecf5e63a033dabe5/src/libraries/Microsoft.NETCore.Platforms/src/Microsoft.NETCore.Platforms.csproj#L45